server/models.js

```diff
@@ -5,4 +5,11 @@ class productDetails {
   constructor() {
     this.model = productDetailsModel;
   }
-}
\ No newline at end of file
+
+  async getProduct(productId) {

```

Comment:
The way you're using async/await is a little funky here

Reply:
This was the first time I tried using it, just wanted to try getting it to work. Out of curiosity what is funky about it.

Reply:
I take it back, it looks fine 👍I wasn't sure how you implemented `findOne` but now that I have the context it makes sense :)